apps/dashboard/app/(app)/semantic-cache/page.tsx

```diff
@@ -0,0 +1,55 @@
+import { PageHeader } from "@/components/dashboard/page-header";
+import { Separator } from "@/components/ui/separator";
+
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+
+import { getAllSemanticCacheLogs } from "@/lib/tinybird";
+
+export default async function SemanticCachePage() {
+  const { data } = await getAllSemanticCacheLogs({});
```

Comment:
this will blow up quickly if we don't put a limit on it :)

Reply:
Done